src/app/RecordingList/ActiveRecordingsList.tsx

```diff
@@ -73,6 +74,13 @@ export const ActiveRecordingsList: React.FunctionComponent<ActiveRecordingsListP
     'State',
   ];
 
+  enum NotificationCategory {
+    RecordingCreated = 'RecordingCreated',
+    RecordingDeleted = 'RecordingDeleted',
+    RecordingSaved = 'RecordingSaved',
+    RecordingArchived = 'RecordingArchived'
+  };
+
```

Comment:
Last round of minor changes - could you pull up this enum and the `RecordingNotificationEvent` interface so that they're declared outside of the component, but still within this same file?

Reply:
Done, thanks